### PR TITLE
:musical_score: Add trebleclef command

### PR DESCRIPTION
**Pull request by Metallicow**
:musical_score: Add a Treble clef command.
https://en.wikipedia.org/wiki/Clef
Note:
At the moment this draws the treble clef as a continuous mathematical
function, ... but the code should probably be tweaked somehow in places
to allow for ?thread cuts? as a few points when jumping between
shapes(since there are multiple) is ending up in outer space.

### CODE REVIEW

**Comment by Metallicow** [comment]
@JoshVarga Screenshot
...see what I mean happens with the multiple shapes...
How should stuff like this be best addressed? 
trebleclef_screenie

**Comment by JoshVarga** [comment]
Metallicow,

I was looking for your email address. Can you email me at josh.varga@gmail.com ?

Thanks,
Josh

**Comment by Metallicow** [comment]
@JoshVarga email sent.

File: CREDITS.ini [hunk 1]
```diff
@@ -80,3 +81,7 @@ Name=Jens-Wolfhard Schicke-Uffmann
 GitHub=Drahflow
 BugFixes=true
 
+[Credit_12]
+Name=Emmett Lauren Garlitz - Some Little Sandy Rd, Elkview, West by GOD Virginia
+GitHub=Oll Em
+Quote=I have a nice cherry chess-top(Glass). "But remember, I NEVER played on it."
```
Comment (Metallicow) [inline CREDITS.ini]: @JoshVarga @redteam316 It seems the anti-1st amendment democrat party Hillary/NSA/MIB/CIA/FBI edited/hijacked this. Watch em. Only trust the marshalls with the black shirts or ask them personally.
Now get your chess on!

**Pull request merged**